src/governance.ts

```diff
@@ -273,6 +275,61 @@ export class GovernanceCanister {
     });
   };
 
+  /**
+   * Splits a neuron creating a new one
+   *
+   * @returns newNeuronId
+   * @throws {@link GovernanceError}
+   */
+  public splitNeuron = async ({
+    neuronId,
+    amount,
+  }: {
+    neuronId: NeuronId;
+    amount: ICP;
+  }): Promise<NeuronId> => {
+    const request = toSplitRawRequest({
+      neuronId,
+      amount: amount.toE8s(),
+    });
+
+    const { command } = await this.certifiedService.manage_neuron(request);
+    const response = command[0];
+    console.log("after da manage neuron call");
```

Comment:
some `console.log` to be removed I guess

Reply:
damn! sorry!

Reply:
done!